website/content/en/preview/faq.md

```diff
@@ -140,7 +140,7 @@ Whenever the Fleet API returns an insufficient capacity error for Spot instances
 
 ### Does Karpenter support IPv6?
 
-Yes! Karpenter dynamically discovers if you are running in an IPv6 cluster by checking the kube-dns service's cluster-ip. When using an AMI Family such as `AL2`, Karpenter will automatically configure the EKS Bootstrap script for IPv6. Some EC2 instance types do not support IPv6 and the Amazon VPC CNI only supports instance types that run on the Nitro hypervisor. It's best to add a requirement to your Provisioner to only allow Nitro instance types:
+Yes! Karpenter dynamically discovers if you are running in an IPv6 cluster by checking the kube-dns service's cluster-ip. When using an AMI Family such as `AL2`, Karpenter will automatically configure the EKS Bootstrap script for IPv6. Some EC2 instance types do not support IPv6 and the Amazon VPC CNI only supports instance types that run on the Nitro hypervisor. Additionally, Windows nodes do not support IPv6. It's best to add a requirement to your Provisioner to only allow Nitro instance types:
```

Comment:
Can we make this a Windows Support Notice like the other ones throughout the docs?